src/components/typography/typography.stories.mdx

```diff
@@ -34,97 +35,7 @@ It's possible to change the text position, using the spacing props. See the prop
-      <Typography variant="h1" color="blackOpacity74">
-        Black Opacity 74
-      </Typography>
-      <Typography variant="h1" color="blackOpacity65">
-        Black Opacity 65
-      </Typography>
-      <Typography variant="h1" color="blackOpacity55">
-        Black Opacity 55
-      </Typography>
-      <Typography variant="p">
-        This is standard text, it is the default variant if you do not supply a
-        variant prop. It has no special importance, but it does have a default
-        margin bottom. You have to provide your own margin and padding to all
-        other variants.
-      </Typography>
-      <Typography variant="p">
-        If you want to{" "}
-        <Typography variant="b">draw attention to content</Typography>, and that
-        content has the same importance as standard text you should use the "b"
-        variant.
-      </Typography>
-      <Typography variant="strong" display="block" mb={1}>
-        Only when the text is more important should you use the strong variant.
-      </Typography>
-      <Typography variant="p">
-        When you want to <Typography variant="em">stress emphasis</Typography>{" "}
-        use the "em" variant. If you are using the variant for styling purposes{" "}
-        <Typography variant="em" as="i">
-          ensure you override the element
-        </Typography>
-      </Typography>
-      <Typography variant="small" display="block" mb={1}>
-        The small variant renders a small elemevant, which is used for small
-        print. The small variant also has a smaller appearance.
-      </Typography>
-      <Typography variant="big" display="block" mb={1}>
-        The big variant uses larger font-face to draw attention but content has
-        the same importance as standard text.
-      </Typography>
-      <Typography variant="p">
-        The 1<Typography variant="sup">st</Typography>, 2
-        <Typography variant="sup">nd</Typography> are examples of superscript.
-      </Typography>
-      <Typography variant="p">
-        H<Typography variant="sub">2</Typography>O is an example of subscript
-      </Typography>
-      <List>
-        <ListItem>
-          Milk <Typography variant="b">2L</Typography>{" "}
-          <Typography variant="em">Skimmed</Typography>
-        </ListItem>
-        <ListItem>
-          Bread <Typography variant="b">500g</Typography>
-        </ListItem>
-        <ListItem>
-          Sugar <Typography variant="b">1Kg</Typography>
-        </ListItem>
-      </List>
-      <List as="ol">
-        <ListItem>
-          Milk <Typography variant="b">2L</Typography>{" "}
-          <Typography variant="em">Skimmed</Typography>
-        </ListItem>
-        <ListItem>
-          Bread <Typography variant="b">500g</Typography>
-        </ListItem>
-        <ListItem>
-          Sugar <Typography variant="b">1Kg</Typography>
-        </ListItem>
-      </List>
-    </>
-  </Story>
+  <Story name="variants" story={stories.VariantsStory} />
```

Comment:
I'm not sure if it was deliberate or not to remove the `parameters={{ info: { disable: true }}`? I'm not sure what it does exactly but it seems to hide the props tables inside the "controls" tab in the addons. Not sure if that's good or not but we should probably be consistent and not change it here - unless there's a good reason for it that I'm missing!

(Note also, after recent experience on something similar, it might not work to leave it here in JSX and you might need to add it in the `.tsx` file as `VariantStory.parameters = { info: { disable: true }}` and the same with the other story.)